stdlib/gc.ml

```diff
@@ -38,6 +38,7 @@ type control = {
   mutable max_overhead : int;
   mutable stack_limit : int;
   mutable allocation_policy : int;
+  window_size : int;
```

Comment:
> If I could add a "deprecated" annotation on these mutable flags

This is tempting!  That would probably be an annotation on the field (e.g. `[@deprecated_mutable]`).